test/test_backend/db/test_utils.py

```diff
@@ -33,10 +35,10 @@ def setUpClass(cls):
     @classmethod
     def tearDownClass(cls):
         """tearDown"""
-        run_sql(cls.con, f"DELETE FROM fetch_audit WHERE comment = 'Unit testing.';")
+        run_sql(cls.con, f"DELETE FROM fetch_audit WHERE comment LIKE 'Unit testing.%';")
```

Comment:
Edge case fix: It might be possible for this not to delete rows as intended, because `comment` field can get updated (e.g. by adding results of fetch resolution).